.github/workflows/fuzz_test.yml

```diff
@@ -100,7 +100,7 @@ jobs:
         if: ${{ failure() }}
         uses: actions/upload-artifact@v3
         with:
-          name: crash_s3_${{ matrix.fuzz-targets }}_${GITHUB_REF}_${GITHUB_RUN_NUMBER}_${$EPOCHSECONDS}
+          name: crash_s3_${{ matrix.fuzz-targets }}_${{ github.ref }}_${{ github.run_attempt }}_${EPOCHSECONDS}
```

Comment:
Where is `EPOCHSECONDS` from? We can remove this if we already have `github.run_attempt`.

Reply:
EPOCHSECONDS is a linux environment variable. I may need a `echo` to join the string. I think `github.run_attempt` is enought

Reply:
I think crash_memory_${{ matrix.fuzz-targets }}_${{ github.ref }}_${{ github.run_attempt }}_${{ github.sha }} is enough